packages/manager/modules/nutanix/src/routing.js

```diff
@@ -14,12 +20,57 @@ export default /* @ngInject */ ($stateProvider) => {
             : false,
         ),
     resolve: {
-      clusters: /* @ngInject */ (NutanixService) =>
+      ...ListLayoutHelper.stateResolves,
+      staticResources: () => true,
+      resources: /* @ngInject */ (NutanixService) =>
         NutanixService.getClusters(),
-      nodeDetails: /* @ngInject */ (clusters, NutanixService) =>
-        NutanixService.getServer(clusters[0].getFirstNode()),
-      getClusterDetailsState: /* @ngInject */ ($state) => (serviceName) =>
-        $state.href('nutanix.dashboard.general-info', {
+      nodeDetails: /* @ngInject */ (resources, NutanixService) =>
+        NutanixService.getServer(resources[0].getFirstNode()).then((data) => {
+          resources[0].setNodeDetails(data);
+        }),
+      apiPath: () => '/nutanix',
+      schema: /* @ngInject */ ($http) =>
+        $http.get('/nutanix.json').then(({ data }) => data),
+      dataModel: () => 'nutanix.cluster',
+      defaultFilterColumn: () => 'serviceName',
+      header: () => 'nutanix',
+      customizableColumns: () => true,
+      columns: /* @ngInject */ ($translate) => {
+        return [
+          {
+            title: $translate.instant('nutanix_cluster_list_name'),
+            property: 'serviceName',
+            sortable: 'asc',
+            searchable: true,
+            filterable: true,
+            template: serviceLink,
+          },
+          {
+            title: $translate.instant('nutanix_cluster_list_number_of_nodes'),
+            template: '{{::$row.getNumberOfNodes()}}',
+          },
+          {
+            title: $translate.instant('nutanix_cluster_list_licence'),
+            property: 'targetSpec.license',
+            template:
+              "<span class='text-capitalize' data-ng-bind='::$row.getLicense()'></span>",
+          },
+          {
+            title: $translate.instant('nutanix_cluster_list_status'),
+            template: statusTemplate,
+          },
+          {
+            title: $translate.instant('nutanix_cluster_list_localisation'),
+            template:
+              "{{:: $row.nodeDetails.datacenter + ' (' + $row.nodeDetails.rack + ')'}}",
```

Comment:
How is't possible to access to $row from routing file?
It's accessible only from HTML component, no ?

Reply:
This uses `ListLayout` from `@ovh-ux/manager-ng-layout-helpers` which has template for `data-grid` view.
For this we can customise columns and pass custom templates to the `data-grid`. This is what is happening here. I am passing the template which will be executed inside data-grid of `ListLayout`

Reply:
Thanks